newsfragments/1123.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix cancellation issues by decoupling cancellation from multi errors.
```

Comment:
Changelog is for users, so we should have something less technical like `Fix internal behavior involving cancelled tasks that could lead to unhandled errors logs`

Reply:
:+1: